packages/cli/src/lib/commands/apps-util.ts

```diff
@@ -106,10 +106,20 @@ const availableScopes = [
 	'w:installedapps',
 ]
 
-export const oauthAppScopeDef = checkboxDef<string>('Scopes', availableScopes)
+export const oauthAppScopeDef = checkboxDef<string>('Scopes', availableScopes, {
+	helpText: 'More information on OAuth 2 Scopes can be found at:\n' +
+	'  https://www.oauth.com/oauth2-servers/scope/\n\n' +
+	'To find out which scopes you need for the application, see documentation for the individual endpoints you will use in your app:\n' +
```

Comment:
Consider using "determine" instead of "find out".

Reply:
Definitely! Fixed.